bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -153,7 +172,24 @@ public void setLocation(JsonObject responseJson) {
                     gpsLongitude = attributes.get("gpsLongitude").getAsDouble();
                 }
                 if (attributes.get("lastUpdateTime") != null) {
-                    locationUpdated = attributes.get("lastUpdateTime").getAsString();
+                    try {
+                        locationUpdated = ZonedDateTime.parse(attributes.get("lastUpdateTime").getAsString());
+                    } catch (DateTimeParseException e) {
+                        logger.warn("Error updating location updated timestamp. {}", e.getMessage());
```

Comment:
Log level should be reduced to **debug**. Otherwise users will see this for every poll cycle and all refresh commands in case the service would start to return incompatible values.

Reply:
Additionally, should `locationUpdated` be set to `null` in this case?

Reply:
Done

Reply:
I set these to null now.  The channel will not get updated if the values are null.  So the last update in the Item value will be the last successful update.